### DISCUSSION TITLE
[BUG] Issue and PR button say Unsubscribe when notifications set for Releases only

### DISCUSSION THREAD

**Discussion by amadejpapez** [community/community#9509] [Mobile] [answered]
When repository notifications are set to Custom - Releases to get notifications only for new releases, all Issue and PR buttons turn to Unsubscribe even though you are not subscribed and don't get notifications for them.

![image](https://user-images.githubusercontent.com/63184600/147851763-ac6a6da6-a09c-4ed5-9440-776114be746f.jpeg)

**Comment by brianlovin** [comment, marked as answer]
@amadejpapez thanks for the report. Could you confirm that when you visit this issue on the GitHub website that you see a different action in the sidebar, here:

<img width="303" alt="Screen Shot 2022-01-04 at 16 25 06@2x" src="https://user-images.githubusercontent.com/1923260/148141722-b697ea60-4741-4aaf-a26c-78472bba9478.png">

Issue and pull request subscribe/unsubscribe buttons aren't always linked to the overall subscription settings for the repository. For example if you created this issue, or participated in it by leaving a comment or being mentioned, then you will be subscribed to future updates for that specific issue until you unsubscribe.

**Reply by amadejpapez** [reply]
It is not because I was involved with that Issue. I just went to a random repository, set notifications to Custom - Releases and now all PR and Issue buttons say Unsubscribe.

If I set it to Custom - PRs it says Unsubscribe on Issues too. With Custom - Issues it says Unsubscribe also on PRs.

I don't receive notifications. Just the Unsubscribe/Subscribe text seems to display incorrectly with Custom notifications in the iOS app.

As you suggested, I also went to the Web version and it says correctly Subscribe.

![image](https://user-images.githubusercontent.com/63184600/148142777-31bc86d4-a7af-428a-bdcb-83487b374594.jpeg)

![image](https://user-images.githubusercontent.com/63184600/148142789-943293ab-d2d8-47e4-9ae4-a029b2a1e4ac.jpeg)

Can you reproduce this?

**Reply by brianlovin** [reply]
Yes, I can reproduce! Flagging for the team, we'll get a fix for this prioritized — thank you!

**Reply by amadejpapez** [reply]
Are there any updates on this issue? It is still happening :)

**Reply by brianlovin** [reply]
No updates yet, but thanks for checking in again — I've followed up with the team.

**Reply by amadejpapez** [reply]
Looks like this is fixed with today's update 🥳
